scripts/fully_covered_files.py

```diff
@@ -0,0 +1,189 @@
+# Copyright 2019 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Array containing the file names of the frontend files that are fully
+covered. It's a whitelist for frontend test coverage."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+fully_covered_files_array = [
+  'MultipleChoiceInput.ts',
```

Comment:
For ease of reviewing/debugging, can we please have this list to be sorted alphabetically? (You can probably print the sorted array and paste it here).

Reply:
Done